account/backend.auto.tfvars

```diff
@@ -1,4 +1,4 @@
-bucket         = "012345678912-us-east-1-tf-state"
+bucket         = "784384215106-us-east-1-tf-state"
```

Comment:
unsure about this, do we want the example to point to an actual account/bucket?

Reply:
For dev purposes I do. It's mostly me doing the dev so I pointed it to my account

Reply:
i mean I guess if it's your account and you don't care I don't have any complaints